README.md

```diff
@@ -25,34 +25,33 @@ MSBuild 15.9 builds from the branch [`vs15.9`](https://github.com/Microsoft/msbu
 | **Full Framework** |[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_Windows_NT_Full)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_Windows_NT_Full)| N/A | N/A | N/A |
 |**.NET Core**|[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_Windows_NT_CoreCLR)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_Windows_NT_CoreCLR)|[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_Ubuntu16.04_CoreCLR)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_Ubuntu16.04_CoreCLR)|[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_OSX10.13_CoreCLR)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_OSX10.13_CoreCLR)|
 
-### Source code
-
-* Clone the sources: `git clone https://github.com/Microsoft/msbuild.git`
-
 ## Building
 
-### Building MSBuild with Visual Studio 2019
+### Building MSBuild with Visual Studio 2019 on Windows
 
 For the full supported experience, you will need to have Visual Studio 2019 or higher.
 
 To get started on **Visual Studio 2019**:
 
 1. [Install Visual Studio 2019](https://www.visualstudio.com/vs/).  Select the following Workloads:
-  - _.NET desktop development_
-  - _.NET Core cross-platform development_
-2. [Install the .NET Core 2.1 SDK](https://www.microsoft.com/net/learn/get-started/windows).
-2. Clone the source code (see above).
-2. Open a `Developer Command Prompt for VS 2019` prompt.
-3. Build the code using the `build.cmd` script at the root of the repo. This also restores packages needed to open the projects in Visual Studio.
-5. Open `MSBuild.sln` or `MSBuild.Dev.sln` in Visual Studio 2019.
+  - .NET desktop development
+  - .NET Core cross-platform development
```

Comment:
I'm not sure whether we should continue to list these after #4676, but we can always change the readme then.